src/extension/features/accounts/toggle-transaction-filters/index.js

```diff
@@ -40,6 +40,10 @@ export class ToggleTransactionFilters extends Feature {
     return true;
   }
 
+  invoke() {
+    //
+  }
+
```

Comment:
re: discussion in slack - can we update the `shouldInvoke` for all of these to return if we're on the appropriate page and then actually update `invoke` 

fwiw, this one should probably return false in `shouldInvoke` since there's nothing happening in `invoke`